README.md

```diff
@@ -1,16 +1,42 @@
 # JDBC Connector
 
-[![Build Status](https://travis-ci.com/modakanalytics/jdbc.almaren.svg?token=TEB3zRDqVUuChez9334q&branch=master)](https://travis-ci.com/modakanalytics/jdbc.almaren)
+[![Build Status](https://github.com/music-of-the-ainur/jdbc.almaren/actions/workflows/Jdbc.almaren-githubactions.yml/badge.svg)](https://github.com/music-of-the-ainur/jdbc.almaren/actions/workflows/Jdbc.almaren-githubactions.yml)
```

Comment:
Please raise another PR, music-of-the-ainur -> modakanalytics. According to the master repo this URL as to be present.